webrtc-stats.html

```diff
@@ -1940,6 +1951,15 @@ <h2>
                   statistics were generated by the local endpoint.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>headerBytesSent</code></dfn> of type <span class=
+                "idlMemberType">unsigned long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP header and padding bytes sent for this SSRC.
```

Comment:
Want to add a comment saying that this + bytesSent = the number of bytes sent as UDP payload (that is, this doesn't include UDP headers or IP headers)

Reply:
There could be another alternative: totalBytesSent, and to get headers and padding you'd subtract bytesSent (which is payload).

I prefer the headerBytes, though.

Reply:
Suggestion: "Total number of RTP header and padding bytes sent for this SSRC. This does not include the size of transport layer headers such as IP or UDP. headerBytesSent + bytesSent equals the number of bytes sent as payload over the transport."